doc/examples/tornado_change_stream_example.py

```diff
@@ -109,10 +109,8 @@ def main():
     try:
         loop.start()
     except KeyboardInterrupt:
-        pass
-    finally:
         if change_stream is not None:
-            change_stream.close()
+            loop.add_callback(change_stream.close)
```

Comment:
Does this actually work? I would expect close to never run since the loop is not running at this point. What if we use loop.run_sync similar to asyncio.run?:

```
    loop = tornado.ioloop.IOLoop.current()
    # Start watching collection for changes.
    try:
        loop.run_sync(lambda: watch(collection))
    except KeyboardInterrupt:
        pass
```